### PR TITLE
[Feat] Added 2 new HW configs and fixed install for dai.yml and voltd…

### PR DESCRIPTION
**Pull request by paul-amonson**
…b.yml.

Signed-off-by: Paul Amonson <paul.d.amonson@intel.com>

### CODE REVIEW

File: docker-deploy/hw2-config/MachineConfig.json [hunk 1]
```diff
@@ -0,0 +1,482 @@
...
+    },
+    {
+      "Parms": null,
+      "TypeOfAdapter": "INVENTORY",
+      "NotifyWhenFinished": "F",
+      "WorkToBeDone": "HandleInputFromExternalComponent",
+      "Queue": "X3001-AM2"
+    },
+    {
+      "Parms": "RabbitMQHost=sms01-nmn.local",
```
Comment (sindhu-106) [inline docker-deploy/hw2-config/MachineConfig.json:164]: "sms01-nmn.local" --> "am02-nmn.local"

File: docker-deploy/hw3-config/MachineConfig.json [hunk 2]
```diff
@@ -0,0 +1,482 @@
...
+    {
+      "Value": "/opt/ucs/etc/log4j2.xml",
+      "Key": "UcsLog4jConfigurationFile"
+    },
+    {
+      "Value": "/opt/ucs/etc",
+      "Key": "UcsConfigDirectory"
+    },
+    {
+      "Value": "am02-nmn.local",
```
Comment (sindhu-106) [inline docker-deploy/hw3-config/MachineConfig.json:16]: should this be "am03" ?

File: docker-deploy/hw3-config/MachineConfig.json [hunk 3]
```diff
@@ -0,0 +1,482 @@
...
+    {
+      "LogFile": "$UCSLOGFILEDIRECTORY/AdapterUI-$LCTN-$INSTANCE.log",
+      "ServiceNode": "X3001-AM2",
+      "Invocation": "com.intel.dai.ui.AdapterUIRest",
+      "TypeOfAdapter": "UI",
+      "NumberOfInstances": 1
+    },
+    {
+      "LogFile": "$UCSLOGFILEDIRECTORY/AdapterRasForeignBus-$LCTN-$INSTANCE.log",
+      "ServiceNode": "X3001-AM2",
```
Comment (sindhu-106) [inline docker-deploy/hw3-config/MachineConfig.json:37]: should this be "X3001-AM3" ?

File: docker-deploy/hw3-config/MachineConfig.json [hunk 4]
```diff
@@ -0,0 +1,482 @@
...
+    },
+    {
+      "Parms": "IntvlBtwnPurgesMs=3600000,AddtlTimeToKeepMovedDataBeforePurgeMs=86400000,RabbitMQHost=am02-nmn.local",
+      "TypeOfAdapter": "ONLINE_TIER",
+      "NotifyWhenFinished": "F",
+      "WorkToBeDone": "DataMover",
+      "Queue": ""
+    },
+    {
+      "Parms": "RabbitMQHost=am02-nmn.local",
```
Comment (sindhu-106) [inline docker-deploy/hw3-config/MachineConfig.json:129]: am03 ?

**Review by sindhu-106 — APPROVED** [review]

**Review by yhuan47 — APPROVED** [review]

**Pull request merged**